squire_sdk/Cargo.toml

```diff
@@ -63,6 +63,8 @@ headers = { version = "0.4", optional = true }
 # To be moved
 hashbag = { version = "0.1.11", features = ["serde"] }
 derive_more = "0.99.17"
+hyper = "1.0"
```

Comment:
The history of this one is a bit weird. Within the history of `development`, we see `hyper = "0.14"` added and removed.

Should this line be here in this PR?

Reply:
Ya, the history is a bit weird, but this line is fine. This change is needed regardless.